types/splitio.d.ts

```diff
@@ -918,8 +918,16 @@ declare namespace SplitIO {
    * Evaluation options object for getTreatment methods.
    */
   type EvaluationOptions = {
+    /**
+     * Whether the evaluation/s will track impressions or not.
+     *
+     * @defaultValue `false`
+     */
+    impressionsDisabled?: boolean;
```

Comment:
We should delete this TypeScript definition, if the feature is not supported yet for SDKs.